optuna/trial/_trial.py

```diff
@@ -469,6 +469,10 @@ def objective(trial):
         Raises:
             :exc:`NotImplementedError`:
                 If trial is being used for multi-objective optimization.
+            :exe:`ValueError`:
+                If ``step is negative`` is specified.
```

Comment:
How about simplifying the sentence?
```suggestion
                If ``step`` is negative.
```

Reply:
I also think I should revise the wording that way.